src/TaggingServiceProvider.php

```diff
@@ -113,7 +113,7 @@ protected function indexTags(): void
      */
     protected function getTagsFromDocBlock(string $docBlock): array
     {
-        $regex = '#(@tag\s*(?P<tags>[^\s]+))#';
+        $regex = '#^\s*/?\**\s*(@tag\s*(?P<tags>[^\s]+))#m';
```

Comment:
I guess, the `m` modifier means that if the tag spans multiple lines, this would still work, right? Although, I don't see how that could be: a tag name is a service name, and should not contain spaces.

Reply:
Actually, the `m` modifier simply changes the `^` and `$` tokens to match the start/end of the line, rather than the start/end of the string ([source](https://www.php.net/manual/en/reference.pcre.pattern.modifiers.php)). The modifier doesn't really change anything. If they can span multiple lines with the modifier, they can span multiple lines without it.

As far as I can tell, new lines are only possible where the `\s` whitespace tokens are, and the only one that I see that would affect tag detection is the token between the `@tag` sequence and the service name capture group. Though in most sensible cases, if the `@tag` string is on a separate line from the service name, there's probably going to be asterisks in between them anyway, so it would fail to match with the regex pattern.

```php
/**
 * @tag
 * my_service
 */
```

You'd have to go out of your way to accommodate it, which now that I look at it makes it seem almost reasonable that it would match with the regex:

```php
/**
 * @tag
    my_service
 */
```

I personally wouldn't lose sleep over this 😅

Reply:
Thank you kindly, sir, for the explanation! 🙏 That is exactly the kind of case I had in mind. And yes, if I really do go out of my way so far in order to do this, then I probably meant to do exactly this for whatever reason - maybe to satisfy a `phpcs` line length limitation with minimal casualties.

So, ya, I pronounce this good and done.